content/00.front-matter.md

```diff
@@ -11,15 +11,15 @@ on {{date}}.
 {% for author in authors %}
 + **{{author.name}}**<br>
   {%- if author.orcid is defined %}
-    ![ORCID icon](images/orcid.svg){height="13px"}
+    ![ORCID icon](images/orcid.svg){height="13px" width="13px"}
     [{{author.orcid}}](https://orcid.org/{{author.orcid}})
   {%- endif %}
   {%- if author.github is defined %}
-    · ![GitHub icon](images/github.svg){height="13px"}
+    · ![GitHub icon](images/github.svg){height="13px" width="13px"}
     [{{author.github}}](https://github.com/{{author.github}})
   {%- endif %}
   {%- if author.twitter is defined %}
-    · ![Twitter icon](images/twitter.svg){height="13px"}
+    · ![Twitter icon](images/twitter.svg){height="13px" width="13px"}
```

Comment:
@vsmalladi would specifying `width="13px"` for these SVGs be valuable independent of the rest of the changes in this PR? These SVGs export to DOCX currently, just they are misshapen?

If these changes are valuable alone, you should open a quick PR with just those changes.

Reply:
@dhimmel I will test them out based on the updates and see if works.